### ISSUE TITLE
Trucks ignoring P2 export request

### ISSUE THREAD

**Issue by mnoreke** [MaFi-Games/Captain-of-Industry-issues#398]
**Describe the issue**
My city trash is set to P2 and is overflowing. Trucks are running other jobs instead of the highest priority I have set.

**Expected behavior**
Treash to be emptied from the storage in town.

**To Reproduce**
1. Load attached save [P2 ignored for trash.zip](https://github.com/MaFi-Games/Captain-of-Industry-issues/files/8949057/P2.ignored.for.trash.zip)
2. Watch the storage outside town be ignored

**Additional context**
 - Game version: 0.4.5a

**Attachments**
Save file is attached

**Comment by utdrmac** [comment]
I found this to be true as well with the recent 0.4.5 update. I believe the issue is that if the mine control tower(s) that accept waste are linked from other sources, then the trucks believe there is nowhere to dump waste. You cannot link the housing waste building to a mine tower. I solved this issue by making a storage bucket nearby and belting the waste into the bucket. Then I linked this bucket to the tower and trucks started taking trash again out of the bucket.
